### PR TITLE
Update admin service eric 24

### PR DESCRIPTION
**Pull request by RoEric**
-"Admin approval service" has been renamed to "admin review service" with submission approval being the current focus of this service
- Updated the entities in the service to be specific to the "approval" function rather than for a general admin service
- Approval only requires the 'Capture' model, so the only property in the service is a collection of Captures
- Next step is to add basic methods to the repository interface to access and interact with the data

### CODE REVIEW

**Comment by RoEric** [comment]
The Admin Review Services has been separated into two different interfaces, IAdminReviewService and IAdminReviewRepository. Implementations for both have been added

**Review by weberwilliamb — APPROVED** [review]
I think this looks good.

**Review by sruble — APPROVED** [review]
Looks good Eric.

**Review by Sara-webaplication — APPROVED** [review]

File: photo-points/Services/AdminReviewServices.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+        public AdminReviewServices(IAdminReviewRepository capturerepo)
+        {
+            _capturerepo = capturerepo;
+        }
+
+        public IQueryable<Capture> GetAllCaptures()
+        {
+            return _capturerepo.GetCaptures();
+        }
```
Comment (Bansenauer-Cascadia) [inline photo-points/Services/AdminReviewServices.cs]: Also include a service that returns all of the captures not yet approved as an IEnumerable<Capture>

**Review by Bansenauer-Cascadia — APPROVED** [review]

File: photo-points/Services/AdminReviewServices.cs (hunk 1 above)
Comment (RoEric) [reply photo-points/Services/AdminReviewServices.cs]: Should be added now in my latest commit

**Comment by RoEric** [comment]
New pull request here:
https://github.com/Cascadia-Connections/photo-points/pull/36

**Pull request closed without merging**